lib/article_json/export/amp/elements/embed.rb

```diff
@@ -4,22 +4,16 @@ module AMP
       module Elements
         class Embed < Base
           include Shared::Caption
+          attr_reader :amp_library
```

Comment:
what do you think about implementing this as a method:
```ruby
def amp_library
  return @amp_library if defined? @amp_library
  type_specific_node # the execution will determine needed amp library
  @amp_library
end
```

like this we would keep `#type_specific_node` private and the logic of when/how stuff is calculated withing this method...

Reply:
I like it, changing it